CONTRIBUTING.md

```diff
@@ -51,4 +52,4 @@ Please follow the steps below in order to make the changes:
    mvn clean install 
    ```
 
-10. Make sure that `example` projects are compiling and running.
+11. Make sure that `example` projects are compiling and running.
```

Comment:
How do you do this?

Reply:
ci

Reply:
install library and plugin and then compile example